src/app/recover-funds/[address]/RecoverFunds.tsx

```diff
@@ -44,37 +43,16 @@ async function getData(
   }
 }
 
-const RecoverFunds = ({ address }: { address: string }) => {
+type Props = {
+  operationInfo: OperationInfo;
+  address: string;
+};
+const RecoverFunds = ({ operationInfo, address }: Props) => {
   const { chain } = useNetwork();
-  const [operationInfo, setOperationInfo] = useState<OperationInfo | null>(
-    null,
-  );
-  const [destinationAddress, setDestinationAddress] = useState<string | null>(
-    null,
-  );
-
   const targetChainID = getTargetChainId(chain?.id);
 
-  useEffect(() => {
-    if (!targetChainID) {
-      return;
-    }
-
-    getData(targetChainID, address).then((data) => {
-      setOperationInfo(data);
-    });
-  }, [address, targetChainID]);
-
-  const handleChange: React.ChangeEventHandler<HTMLInputElement> = (e) => {
-    const value = e.target.value;
-    setDestinationAddress(value);
-  };
-
   // No funds to recover
-  if (
-    operationInfo &&
-    !hasBalanceOverThreshold(operationInfo.balanceToRecover)
-  ) {
+  if (!hasBalanceOverThreshold(operationInfo.balanceToRecover)) {
     return (
       <div className="funds-message">
         There are no funds stuck on {operationInfo.aliasedAddress} (Alias of{' '}
```

Comment:
The addresses are long making this line very long. Add a linebreak before "(Alias...)"?